static/stable/stage/navigation/openshift-navigation.json

```diff
@@ -17,16 +36,6 @@
             "title": "Learning Resources",
             "href": "/openshift/learning-resources"
         },
-        {
-            "id": "overview",
-            "appId": "openshift",
-            "href": "/openshift/overview",
-            "title": "Overview",
-            "icon": "OpenShiftIcon",
```

Comment:
@Randy424 I'm not sure what this `icon` property is used for, but you did remove it. Do you know?

Reply:
@KevinFCormier The icons are a new addition to the Services dropdown menu in Chrome.